.github/workflows/workflow.yml

```diff
@@ -6,6 +6,20 @@ on:
   pull_request:
     types: [opened, synchronize, reopened]
 jobs:
+  # Job to run change detection
+  changes:
+    runs-on: ubuntu-latest
+    # Set job outputs to values from filter step
+    outputs:
+      buildtest: ${{ steps.filter.outputs.buildtest }}
+    steps:
+      # For pull requests it's not necessary to checkout the code
+      - uses: dorny/paths-filter@v2
+        id: filter
+        with:
+          filters: |
+            buildtest:
+              - 'packages/build/**'
```

Comment:
if you go the route with that you need to include things like:

```
package-lock.json
nx.json
ava.base.js
```

+ changes to the testing repo or builds dependencies

you can checkit out with `npx nx graph`.

That's the reason why I would have tried doing it with `npx nx print-affected`

Reply:
Looking at the dependency graph below, looks like we will always have to run the **test-build** package if any of the 7 out of 8 packages are modified. Statistically speaking, we will always have to run the **test-build** pipeline. Proposing to close this PR.

<img width="1493" alt="Screenshot 2022-10-11 at 10 14 50" src="https://user-images.githubusercontent.com/12011447/195050578-9ed86564-c24b-454f-85bb-749f11d2b0e1.png">